### ISSUE TITLE
Use release

### ISSUE THREAD

**Issue by samuelds** [TommyMaes/SetVol#2]
Hi, 

Can you use release for download last version ? 

Not good practice push zip file on git project :D

**Comment by TommyMaes** [comment]
This is done on purpose, the program is not perfectly written and i have had very specific occasions where, if donwloaded and installed from within a specific temp folder (while being executed as admin or something along that line),  it would copy the entire C drive to itself. To prevent this i packed it all into a RAR where there is little chance to unpack in or install from those specific temp folders

In short: this is done so people don't break there computer

Its not a bug, its a feature... to disguise a bug...
